test/distributed/org/apache/cassandra/distributed/util/Auth.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.util;
+
+import org.apache.cassandra.auth.CassandraRoleManager;
+import org.apache.cassandra.distributed.api.IInvokableInstance;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
+public class Auth
+{
+
+    public static void waitForExistingRoles(IInvokableInstance instance)
```

Comment:
No existing utility classes this can fit in?  That would be preferably to me, more chance of others finding and using it rather than making their own version of it, if it is in a common location.

Reply:
`CQLTester` imo is a good candidate. My 2cts.

Reply:
`CQLTester` is unrelated to in-jvm dtests. There is no utility class in distributed in-jvm tests. I can only move it either Cluster or Instance implementation

Reply:
“waitForAuthInitialization” seems very reasonable to be on the cluster object actually?  Or possibly even have cluster start by default not return until that has happened if auth was enabled?

Reply:
moved to `Cluster`. I don't want to call it automatically because that would introduce some implicit semantic. I didn't really investigated what those tests are doing...

Reply:
`SSTableGeneratorTest` in the distributed package already uses `CQLTester` so it wouldn't be the first case...

Reply:
This is not a good example. That test is a fuzz test implemented with Harry and has nothing to do with in-jvm dtests as far as I can see (except being under the `distributed` root)